tools/azure-sdk-tools/packaging_tools/change_log.py

```diff
@@ -175,6 +179,23 @@ def client(self, old_report, new_report):
             self.breaking_changes.append(msg)
         return
 
+    def compare_operation_function(self):
+        '''
+        Record changelog like rename operation.delete to operation.begin_delete change
+        instead of using remove operation.delete or add operation.begin_delete
```

Comment:
```suggestion
        Record changelog like "rename operation.delete to operation.begin_delete"
        instead of "remove operation.delete or add operation.begin_delete"
```